tests/api/data/test_users_auth_bad_pass.json

```diff
@@ -0,0 +1,11 @@
+{
+    "endpoint": "users/auth",
+    "arguments": {
+        "pass": "foo"
+    },
+    "output": {
+        "error": true,
```

Comment:
This should not be inside the "output" object.
The "error" field tells us that we expect and error and the output is formatted as such,
it is not the actual output
(whereas both the code and message can be considered output).